user/serializers.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.auth import get_user_model, authenticate
+from rest_framework import serializers
+from django.utils.translation import gettext as _
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = get_user_model()
+        fields = ("id", "email", "telegram_nick", "password", "is_staff")
+        read_only_fields = ("is_staff",)
+        extra_kwargs = {"password": {"write_only": True, "min_length": 5}}
```

Comment:
I'm not sure, that we can make password with min_length = 5 because we will get error from DB, because set_password for AbstractUser requiers at least 8 symbols